### PR TITLE
Adding initial package for readme verification.

### PR DESCRIPTION
**Pull request by scbedd**
@Azure/azure-sdk-eng 

`doc-warden` has tested successfully on python 2.7 and 3.6. 

Other bookkeeping:
* Updating root readme

Example usage: https://github.com/Azure/azure-sdk-for-python/pull/4365

### CODE REVIEW

**Comment by weshaggard** [comment]
@scbedd do we expect to setup a CI for this repo as well or will the builds just be manually done? While I want to say setup a CI I worry about what might happen if we start needing toolsets for lots of different languages etc when we add more tools. However I think having CI outweighs that risk, at least for now :)

**Comment by scbedd** [comment]
I actually deliberately placed the package into a `python-packages` folder, so it will be separate from the the others and easy to handle in CI.

If we were to add a NPM package, or C#, or whatever else, we'd put them in a `<language>-packages` folder.

**Comment by bsiegel** [comment]
Looks reasonable to me! :shipit:

File: .azure-pipelines/warden.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+trigger:
+- master
+
+jobs:
+- job: 'Build_Package'
```
Comment (weshaggard) [inline .azure-pipelines/warden.yml]: guidelines call this "Build".

Comment (scbedd) [reply .azure-pipelines/warden.yml]: For sure. Changing.

**Pull request merged**